crates/viewer/re_viewer/src/ui/memory_panel.rs

```diff
@@ -82,20 +82,94 @@ impl MemoryPanel {
-                    ui.collapsing("Primary Query Caches", |ui| {
-                        Self::caches_stats(ui, &store_stats.query_cache_stats);
+                        ui.separator();
+                        ui.collapsing("Viewer Caches", |ui| {
+                            ui.horizontal(|ui| {
+                                ui.label(format!(
+                                    "CPU Memory: {}",
+                                    format_bytes(store_stats.viewer_cache_size as f64)
+                                ));
+                                let gpu_memory = store_stats
+                                    .cache_memory_reports
+                                    .values()
+                                    .map(|report| report.bytes_gpu.unwrap_or_default())
+                                    .sum::<u64>();
+                                if gpu_memory > 0 {
+                                    ui.label(format!(
+                                        "GPU Memory: {}",
+                                        format_bytes(gpu_memory as f64)
+                                    ));
+                                }
+                            });
+                            let mut display_caches =
+                                store_stats.cache_memory_reports.iter().collect::<Vec<_>>();
+
+                            // Iterating a hash map doesn't give us a consistent ordering so we sort by name here.
+                            display_caches.sort_by_key(|(name, _)| *name);
+
+                            for (name, report) in display_caches {
+                                ui.collapsing(*name, |ui| {
+                                    Self::cache_memory_report(ui, name, report);
+                                });
+                            }
+                        });
                     });
-                });
+                }
+            });
+        }
+    }
+
+    fn cache_memory_report(ui: &mut egui::Ui, name: &str, report: &CacheMemoryReport) {
+        ui.horizontal(|ui| {
+            ui.label(format!(
+                "CPU Memory: {}",
+                format_bytes(report.bytes_cpu as f64)
+            ));
+            if let Some(bytes_gpu) = report.bytes_gpu {
+                ui.label(format!("GPU Memory: {}", format_bytes(bytes_gpu as f64)));
             }
+        });
+
+        if !report.per_cache_item_info.is_empty() {
+            egui::ScrollArea::vertical()
+                .max_height(200.0)
+                .id_salt(name)
+                .show(ui, |ui| {
+                    egui::Grid::new(format!("{name} grid"))
+                        .num_columns(3)
+                        .show(ui, |ui| {
+                            ui.label(egui::RichText::new("Name").underline());
+                            ui.label(egui::RichText::new("Memory (cpu)").underline());
+                            ui.label(egui::RichText::new("Memory (gpu)").underline());
```

Comment:
nit: inconsistent to the rest

```suggestion
                            ui.label(egui::RichText::new("Memory (CPU)").underline());
                            ui.label(egui::RichText::new("Memory (GPU)").underline());
```